src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -4969,6 +4969,12 @@ public partial class ObjectDisposedException : System.InvalidOperationException
         public ObjectDisposedException(string? objectName, string? message) { }
         public override string Message { get { throw null; } }
         public string ObjectName { get { throw null; } }
+        [System.Diagnostics.CodeAnalysis.DoesNotReturnAttribute]
+        [System.Diagnostics.StackTraceHidden]
```

Comment:
`StackTraceHidden` should not be in the reference assembly. It should be just in the implementation.